source/Outfit.cpp

```diff
@@ -196,6 +196,10 @@ void Outfit::Load(const DataNode &node)
 				if(isNewLicense(licenses, grand.Token(0)))
 					licenses.push_back(grand.Token(0));
 		}
+		else if(child.Token(0) == "inability" && (child.HasChildren() || child.Size() >= 2))
```

Comment:
Not sure `inability` is an appropriate name for the `permissions` on an outfit (or ship). This should also perhaps support `remove`.